vllm/worker/tpu_model_runner.py

```diff
@@ -601,11 +598,32 @@ def _execute_model(*args):
         return [SamplerOutput(sampler_outputs)]
 
 
-class ModelWrapper(nn.Module):
+class ModelWrapper(TorchCompileWrapperWithCustomDispacther):
 
     def __init__(self, model: nn.Module):
-        super().__init__()
         self.model = model
+        compiled_callable = torch.compile(self.forward,
+                                          backend="openxla",
+                                          fullgraph=True,
+                                          dynamic=False)
+        super().__init__(compiled_callable)
+
+    def __call__(self, *args, is_prompt: bool = False, **kwargs):
```

Comment:
nit: 
```suggestion
    def __call__(self, *args, is_prompt: bool, **kwargs):
```

Reply:
I'd like to avoid implicit errors. Also, it looks a bit weird that there's a case that the model runs prompt while `is_prompt=False`.